datarepo-clienttests/src/main/java/common/utils/BigQueryUtils.java

```diff
@@ -14,6 +15,7 @@
 import java.util.concurrent.atomic.AtomicInteger;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.threeten.bp.Duration;
```

Comment:
I think this is the wrong import, I think you want the java.time one.
```suggestion
import java.time.Duration;
```

Reply:
That's what I thought as well, but it's `org.threeten.bp.Duration`: https://github.com/googleapis/gax-java/blob/main/gax/src/main/java/com/google/api/gax/retrying/RetrySettings.java#L36

Reply:
Well, that's not confusing at all :( Apparently this is a library that provides JDK 8 time support for JDK 6 and 7. It's a shame it's not used in a way that's forward compatible; the code could accept either type and convert it.